app/controllers/verify_controller.rb

```diff
@@ -10,6 +10,7 @@ class VerifyController < ApplicationController
   before_action :confirm_profile_has_been_created, if: :first_step_is_personal_key?
 
   def show
+    session[:email] = 'bruce.wayne@batcave.com'
```

Comment:
We don't want to be dealing with the session in this controller, and we'll also need the "forgot password" flow to actually work with the user's real email. If this is needed for the destination, can we update the destination controller to find the current user's email instead?

Reply:
Sounds good, lets do it in a separate PR to avoid more git rebasing drama.